admin_guide/sysctls.adoc

```diff
@@ -52,7 +52,19 @@ The following sysctls are known to be namespaced:
 - *_kernel.msg*_*
 - *_kernel.sem_*
 - *_fs.mqueue.*_*
-- *_net.*_*
+
+Additionally, most of the sysctls in the *_net.*_* group are known
+to be namespaced. Their namespace adoption differs based on the kernel
+version and distributor.
+
+To check which *_net.*_* sysctls are namespaced on your system, run the
+following command:
+
+----
+$ podman run --rm -ti fedora \
+    /bin/sh -c "dnf install -y findutils && find /proc/sys/" \
+    | grep -e "/proc/sys/net"
```

Comment:
Will got `Error: Failed to synchronize cache for repo 'updates'` with `--net host`.

And better make grep command in the shell command like:
`podman run --net host --rm -ti fedora     /bin/sh -c "dnf install -y findutils && find /proc/sys/ |  grep -e /proc/sys/net"`

Reply:
Another thing is that we better not make any assumption for images,
means better user qualified for container image name:
`podman run --net host --rm -ti docker.io/fedora /bin/sh -c "dnf install -y findutils && find /proc/sys/ | grep -e /proc/sys/net"`

Reply:
You can't use `--net host` as that would make additional sysctls appear.

I don't really see a reason why `--net host` should change anything about the `Error: Failed to synchronize cache for repo 'updates'` message, except for maybe firewall blocking your containers connecting to internets?

I'll add the grep to the `bach -c` command, that's a good improvement, thanks.